docs/data/release_archive.yml

```diff
@@ -5,8 +5,8 @@ release_archive:
       release_date: 2023-10-25
     -
       version_short: "1.17"
-      version_long: 1.17.1
```

Comment:
IIRC, the old version(1.17.1) should not be removed.  See the versions `1.15.3`, `1.15.4`, etc

Reply:
Thanks for the catch!